tools/update_required_branch_checks.py

```diff
@@ -46,6 +46,9 @@
     # Excludes evergreen as blackbox test.
     'evergreen-as-blackbox_test',
 
+    # Excludes long running evergreen_test.
```

Comment:
Are these flaky? They run for longer, but they do catch issues that other tests don't currently and it's more often than not due to an actual failure in my experience. I'd rather keep them.

Reply:
We've talked a lot about this and this test should be moved out of the pre-submits. I'll take that out of here and put it in a separate PR.

Reply:
It should be moved out of pre-submits entirely? Doesn't that mean this doesn't have to be added then?